### PR TITLE
[TESTING] Deploy PREVIEWS integration

### PR DESCRIPTION
**Pull request by SMakaiTakori**
THIS IS A TEST PR. Integration of Netlify was created in this PR.

### CODE REVIEW

**Comment by SMakaiTakori** [comment]
@jeffmaher I have removed the files we talked about and all tests are passing. Netlify deployment is running smoothly as well. All that is needed after this merge will be the finished documentation for how to edit files. Let me know if this looks good or you see anything that needs to be changed. Thanks!

Decisions we need to make for this repo:

- How do we want to update the pull request template for bloom folks 
- Who will the codeowners be for this repo/who should be automatically tagged for the approval of PRs

**Review by jeffmaher — APPROVED** [review]
LGTM!

**Pull request merged**
